.github/workflows/build_and_test.yml

```diff
@@ -1,3 +1,5 @@
+## me fail english? that's unpossible.
+
```

Comment:
please fix this - unmaintainable

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Maintainable

Reply:
looks good - maintainable

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable

Reply:
awesome and elegant - i never would have thought of this

you know Kung Fu. - Neo, paraphrased

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Elegant

Reply:
The software performs an operation at a privilege level that is higher than the minimum level required, which creates new weaknesses or amplifies the consequences of other weaknesses.
TEST

---
**Sema Reaction:** :ok_hand: This code looks good

Reply:
test 6/16/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Tags:** Not reusable, Fault-tolerant

Reply:
test 855

Reply:
test 666

Reply:
test 812

Reply:
test 813

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Inelegant, Fault-tolerant